requirements_test.txt

```diff
@@ -1,5 +1,6 @@
-coverage==3.7.1
```

Comment:
These are pretty outdated. What guideline should be followed to pick new versions?

Reply:
Not sure about that. Maybe we can start from a clean sheet by setting the latest releases and accepting only fixes?

Reply:
Keep in mind these are pinned so that if someone is working on something unrelated, it doesn't start failing all their tests.

So now is a perfect time to bump them... I suggest bumping them all to the latest release and see if it passes. If it does, then we go with it. If not, then either fix or the pin to an older version that works, and file a ticket to update it.

Reply:
I have split up requirements and version constraints. 
So our `requirement_*.txt` files are what we need or what we are (in)compatible with whereas the `constraints.txt` file contains the versions we pin.

Makes sense?

Reply:
After thinking about it more, I'd probably keep them in one file for simplicity, and then actually _not_ do any pinning of test requirements unless we _know_ a particular version's latest release is broken for us.

The worst that happens is a PR starts failing tests until we realize the issue, so it's not that big of a deal. KISS and all that.

And if someone says "hey, this version is broken" but it's not latest, then they can upgrade... unlike production deps, it's not hard to do for test-only deps.

Thoughts?